### PR TITLE
Add Mollie ESlint config and Prettier.

### PR DESCRIPTION
**Pull request by vernondegoede**

### CODE REVIEW

**Comment by lvgunst** [comment]
Have you run the prettier / eslint formatter on the code? No changes?

File: src/models/method.js [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+  static KBC = "kbc";
+  static GIFTCARD = "giftcard";
+  static INGHOMEPAY = "inghomepay";
 
   constructor(props) {
     super(props);
 
     const defaults = {
-      resource: 'method',
+      resource: "method",
```
Comment (lvgunst) [inline src/models/method.js]: I see a lot of changes to use double quotes while you have configured single quotes in `.prettierrc`. We prefer single quotes in all our projects.

Comment (vernondegoede) [reply src/models/method.js]: Yeah, there's something wrong. I ran Prettier again this morning (just to be sure) and it doesn't seem to recognize the `.prettierrc` file. Will check it again later today 👍

**Comment by vernondegoede** [comment]
@lvgunst Can you take a look again? Ran both Prettier + ESLint. Config seems to work now. I changed the package name to `@mollie/eslint-config-node`.

File: src/models/customer.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export default class Customer extends Model {
     super(props);
 
     const defaults = {
-      resource: 'customer',
+      resource: "customer",
```
Comment (lvgunst) [inline src/models/customer.js]: Somehow it's still using double quotes.

File: src/models/customer.js [hunk 3]
```diff
@@ -20,8 +20,8 @@ export default class Customer extends Model {
       _links: {
         mandates: null,
         subscriptions: null,
-        payments: null,
-      },
+        payments: null
```
Comment (lvgunst) [inline src/models/customer.js]: Also the `"trailingComma": "all"` is not working

**Pull request merged**